buildlib/travis-build

```diff
@@ -10,6 +10,7 @@ cd build
 # The goal is warning free compile on latest gcc.
 CC=gcc-6 CFLAGS=-Werror cmake -GNinja ..
 ninja
+../buildlib/check-build --src ..
```

Comment:
Jason, can you please put this call all into .travis.yml?

It will give us clean structure of subscripts for travis.

Thanks